http/rest-client-reactive/src/main/java/com/redhat/BrokenBlockingApi.java

```diff
@@ -0,0 +1,26 @@
+package com.redhat; // this bug can not be reproduced in io.io.quarkus package
```

Comment:
why can it not be reproduced in our packages?

Reply:
I read upstream review, fix PR and also don't get it. The comment needs to be improved to explain circumstances.